client/src/components/Hero.jsx

```diff
@@ -10,42 +15,42 @@ const Hero = () => {
           <FaBriefcase className="inline-block align-bottom" />
         </h1>
         <p className="mt-6 text-lg leading-8 text-gray-600">
-          MatchIQ uses AI to help find your best match (add more in this
-          caption)
+          MatchIQ uses AI to help find the best job for you!<br />
+          <i>Choose from some of these popular searches:</i>
         </p>
       </div>
       <div className="mt-10 text-center">
         <div className="flex flex-wrap gap-6 mb-5 justify-center">
-          <Button color="light" pill>
+          <Button color="light" pill onClick={() => handleClick('front-end')}>
             front-end
           </Button>
-          <Button color="light" pill>
+          <Button color="light" pill onClick={() => handleClick('ux designer')}>
```

Comment:
since they dont do anything if you add an onclick handler its going to throw a bunch of errors unless you initialize empty functions

Reply:
As discussed, this works properly as is